bats/tests/k8s/up-downgrade-k8s.bats

```diff
@@ -70,10 +71,24 @@ verify_images() {
     verify_images
 }
 
+# Remove all the kubectl clients from the .kuberlr directory.
+# Then run `kubectl`, and it should pull in the `kubectl` for
+# the current k8s version in that directory.
+
+verify_kuberlr_for_version() {
+    k8sVersion=$1
```

Comment:
We don't do `camelCase` in bash, we use either all `UPPERCASE`, or `snake_case` for variable and function names.

Also using `local` would be better:

```suggestion
    local k8s_version=$1
```